vignettes/example_models.Rmd

```diff
@@ -33,14 +33,65 @@ Below are a few examples of common statistical models implemented in greta.
 
 <hr>
 
+```{r child='examples/linear_uninformative.Rmd'}
+```
+
+<hr>
+
+```{r child='examples/linear_ridge.Rmd'}
+```
+
+<hr>
+
+```{r child='examples/linear_lasso.Rmd'}
+```
+
+<hr>
+
 ```{r child='examples/multiple_linear.Rmd'}
 ```
 
 <hr>
 
+```{r child='examples/multiple_linear_lasso.Rmd'}
+```
+
+<hr>
+
+```{r child='examples/multiple_linear_warpbreaks.Rmd'}
+```
+
+<hr>
+
+```{r child='examples/multiple_linear_multilogit.Rmd'}
+```
+
+<hr>
+
 ```{r child='examples/hierarchical_linear.Rmd'}
 ```
 
+<hr>
+
+```{r child='examples/hierarchical_linear_slopes.Rmd'}
+```
+
+<hr>
+
+```{r child='examples/hierarchical_linear_general.Rmd'}
```

Comment:
I think these three are a bit more advanced, and should probably go in a separate section (Bayesian neural nets aren't likely to be common models for more first-time greta users!)